src/pages/settings/AddSecondaryLoginPage.js

```diff
@@ -129,30 +129,19 @@ class AddSecondaryLoginPage extends Component {
                     <View style={styles.mb6}>
                         <TextInput
                             label={this.props.translate('common.password')}
-                            value={this.state.password}
-                            onChangeText={password => this.setState({password})}
+                            inputID="password"
                             secureTextEntry
                             autoCompleteType="password"
                             textContentType="password"
-                            onSubmitEditing={this.submitForm}
+                            onSubmitEditing={this.onSubmit}
                         />
                     </View>
                     {!_.isEmpty(this.props.user.error) && (
                         <Text style={styles.formError}>
                             {this.props.user.error}
                         </Text>
                     )}
-                </ScrollView>
-                <FixedFooter style={[styles.flexGrow0]}>
-                    <Button
-                        success
-                        isDisabled={this.validateForm()}
-                        isLoading={this.props.user.loading}
```

Comment:
Since the loading prop is at the user key we won't see the loading indicator at the button. Please address it.

Reply:
Oh I just found we were using wrong key. We don't set the `loading` at `user` we set it at `ONYXKEYS.ACCOUNT` https://github.com/Expensify/App/blob/e434efa1c455620c510441628b94081e0f8a564e/src/libs/actions/User.js#L175

Reply:
Just a question in a case where we are using two different onyx keys for example at set password page we get error from two different keys https://github.com/Expensify/App/blob/e434efa1c455620c510441628b94081e0f8a564e/src/pages/SetPasswordPage.js#L95
so how are we going to display the error at `formState`. Can't we create a prop and just pass the error to `Form` and add it as or like this 


```diff
-   const latestErrorMessage = ErrorUtils.getLatestErrorMessage(this.props.formState);
+   const latestErrorMessage = ErrorUtils.getLatestErrorMessage(this.props.formState) || this.props.errorMessage;
```
https://github.com/Expensify/App/blob/533e63ce8b13e573024784836cef0bb7e859e937/src/components/Form.js#L89
same for loading, can we create a prop `isLoading` and pass that to Form and use it so that we don't have to worry about different keys and formState will still work fine if we use the same key

Reply:
Good question. I am not sure about the answer though. 
cc: @luacmartins (as you have the most knowledge on this topic)

Reply:
I don't think that we should add more props to Form for this situation. Why are we keeping error and loading states in two different keys?

Reply:
No idea.

Reply:
We are also keeping errors in two different keys for set password page so it looks like this is not uncommon 
https://github.com/Expensify/App/blob/e434efa1c455620c510441628b94081e0f8a564e/src/pages/SetPasswordPage.js#L95

So instead of adding a new prop we should be using one key, am I right?

Reply:
While we are at it. Can you please help with accessing the ref of a TextInput which is wrapped with Form component. I wanted to access the ref of email/phone number TextInput so that I can use focus method on it when transition ends.
I am getting undefined.
https://github.com/Expensify/App/blob/d6bac257e4fd1200b5bab27bb157fc312969c14a/src/pages/settings/AddSecondaryLoginPage.js#L99

Reply:
>So instead of adding a new prop we should be using one key, am I right?

Yea, I don't think a form should be storing these in multiple keys.

>Can you please help with accessing the ref of a TextInput which is wrapped with Form component. I wanted to access the ref of email/phone number TextInput so that I can use focus method on it when transition ends.

Is this a requirement? I think it looks bad on mobile when we transition and open the keyboard, especially in smaller devices.

Reply:
> Is this a requirement? I think it looks bad on mobile when we transition and open the keyboard, especially in smaller devices.

We had this implemented and we just added this at some more pages so I think we will need to add this again.

Reply:
Ok, in that case does the `autoFocus` prop work? e.g. https://github.com/Expensify/App/blob/b0cfde3737755cb35d3e5234d36df6c640a650e2/src/components/PasswordPopover/BasePasswordPopover.js#L82

Reply:
Yeah it works but the only problem is that the keyboard will not open after the field gets focused at android unless we use also pass `shouldDelayFocus`

Reply:
Sounds like we have a solution then